octobot_trading/personal_data/orders/order.py

```diff
@@ -312,7 +312,7 @@ def get_computed_fee(self, forced_value=None):
             currency = computed_fee[enums.FeePropertyColumns.CURRENCY.value]
         else:
             symbol_fees = self.exchange_manager.exchange.get_fees(self.symbol)
-            fees = decimal.Decimal(f"{symbol_fees[self.taker_or_maker]}")
+            fees = decimal.Decimal(f"{symbol_fees[self.taker_or_maker]}") / constants.ONE_HUNDRED
```

Comment:
I think we should uniformize at lower level (just after we call ccxt to request fees)

Reply:
Good idea 👍